Gemfile

```diff
@@ -59,6 +59,8 @@ group :development do
   # Use console on exceptions pages [https://github.com/rails/web-console]
   gem 'web-console'
 
+  gem 'rspec-rails'
+
   # Add speed badges [https://github.com/MiniProfiler/rack-mini-profiler]
   # gem "rack-mini-profiler"
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add some more meaningful features in the following sections below ⬇️ to make your readme more descriptive and professional 👍

* 🤩 Key Features
* 🔭 Future Features